### DISCUSSION TITLE
¿Es posible esta nueva opción?

### DISCUSSION THREAD

**Discussion by Eskua** [CarlosMiraGarcia/forocoches-premium#9] [Ideas]
Buenas, enhorabuena por la extensión. La opción de ignorar temas por palabras me ha dado la vida, pero quería preguntarte si es posible (y te parece útil) implementar algo parecido pero al revés, que los temas que contengan determinadas palabras en el título se resalten en algún color para que sea más fácil verlos de un vistazo

**Comment by ghost** [comment]
Buenas.
Si, no ve ningún problema en implementar eso.
Me lo apunto para añadirlo, pero me va a llevar un tiempo ya que ando un poco liado ahora mismo y estoy también haciendo algunos cambios en la extensión que me están llevando más de lo que pensaba.

Muy buena sugerencia!
